synnax/pkg/distribution/ontology/search/search_test.go

```diff
@@ -23,6 +23,7 @@ var _ = Describe("Search", func() {
 		var idx *search.Index
 		BeforeEach(func() {
 			idx = MustSucceed(search.New())
+			idx.Register(ctx, schema.Schema{Type: "test"})
 		})
 		DescribeTable("Term Searching",
```

Comment:
I recall the `[` and `]` characters were problematic, we could add a few tests on those while we're at it?